docs/design/StratisSoftwareDesign.lyx

```diff
@@ -2921,8 +2936,8 @@ Multiple blockdevs being updated with the same metadata must write identical
 \end_layout
 
 \begin_layout Itemize
```

Comment:
Maybe you can leave this out entirely? Software is a broad category, but with upcoming PR, _stratisd_ will be reading both.